contracts/TokenClaim.sol

```diff
@@ -62,4 +67,13 @@ contract TokenClaim is FactoryFriendly, ITokenClaim {
             (VotesToken(parentToken).balanceOfAt(claimer, snapShotId) * parentAllocation) /
             VotesToken(parentToken).totalSupplyAt(snapShotId);
     }
+
+    /// @notice Returns unclaimed tokens after the deadline to the funder.
+    function reclaim() external {
+        require(msg.sender == funder, "caller is not the funder");
+        require(deadline != 0, "no deadline set");
+        require(block.number >= deadline, "deadline has not elapsed");
```

Comment:
Since we're moving to custom error messages instead of require statements, want to go ahead and convert these?

Reply:
Sure, just opened it here: https://github.com/decent-dao/fractal-contracts/pull/40